fickling/loader.py

```diff
@@ -0,0 +1,25 @@
+import pickle
+
+from fickling.fickle import Pickled
+
+
+# TODO Determine what the best defaults for block are
+def load(file, run_after_analysis=True, block=[3, 4, 5]):
```

Comment:
To work seamlessly as a `pickle.load` replacement this function should accept and forward positional and kw arguments that `pickle.load` usually takes (see the [pickle.load() documentation](https://docs.python.org/3/library/pickle.html#pickle.load)).

I think you can do something like
```
def load(file, max_acceptable_severity=SEVERITY.LIKELY_MALICIOUS, *args, **kwargs):
<...>
    return pickle.loads(pickled_data.dumps(), *args, **kwargs)
<...>
```